dist/nearlib.js

```diff
@@ -1252,6 +1256,25 @@ function parseNearAmount(amt) {
     return `${wholePart.add(fractionPart).toString(10, 0)}`;
 }
 exports.parseNearAmount = parseNearAmount;
+/**
+ * Truncates the fraction part of a number to a given number of digits and then trims extra zeroes.
+ * @param value
```

Comment:
why add @param tags with no docs?

Reply:
I removed the fn so it's no longer relevant